pkgs/by-name/mc/mcp-nixos/package.nix

```diff
@@ -0,0 +1,62 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3Packages,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "mcp-nixos";
+  version = "1.0.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "utensils";
+    repo = "mcp-nixos";
+    tag = "v${version}";
+    hash = "sha256-NwP+zM1VGLOzIm+mLZVK9/9ImFwuiWhRJ9QK3hGpQsY=";
+  };
+
+  patches = [
+    ./tests-mock-nix-channels.patch
```

Comment:
I understand that the filename of this patch is self-explanatory, but usually a comment is added to explain why the patch is required. It would be helpful for maintainers for future reviews, to know if they have to remove it or not, without scrutinizing the patch.

Reply:
Added a comment.